CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing Topics to the Show
+
+## Submitting a Conference
+Submit Upcoming Conferences to be talked about on the show using one our [conference issue templates](https://github.com/python-community-news/topics/issues/new/choose).
+
+> NOTE: All Conference announcements should have an issue made. Issues will be closed and not reported on if they fail to meet the criteria for a valid conference, announcement type, or are not approved but the hosts.
+
+### Types of Conferences
+We would love you to submit any conference that would be of interest to the Python Community.
+
+Here are some examples:
+
+1. A Conference _celebrating Python in a geographic region or technical discipline ([PyCon US](https://pycon.us), [PyGotham](https://pygotham.org), [PyData](https://pydata.org/upcoming-events/))
+2. The Conference **Celebrates a Community where Python is widely used** ([TableauCon](https://www.tableau.com/events/conference), [Open Data Science Conference](https://odsc.com))
+3. The Conference is a General Technology Focus with a **Python/Data Science specific track** ([All Things Open](https://www.allthingsopen.org))
+4. A community-run general technology conference ([!!Con](https://bangbangcon.com))
+
+If there is any confusion, **we encourage you to submit it**. One of the maintainers will request more information in the issue comments if necessary.
+
+### Conference Requirements
+- Public CFP (The CFP window may be closed but a CFP process must have taken place)
+- Code of Conduct
+
+### Types of Announcements
+
+The show will report on the following announcements:
+ 
+ 1. Conference Dates Announced
+ 2. Tickets Go On Sale (We will not do any follow up posts regarding discount windows)
```

Comment:
I'd make an exception to this rule, which is that we may partner with conferences to offer PCN-specific discounts where appropriate. No need to change any wording here, just throwing it out there as a call we might make for conferences on a case-by-case basis. Agreed on the "we won't mention early-bird-round-three ticket window" kind of update, though.